tests/Unit/Strategy/StrategyContainerTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+
+namespace Exolnet\Bento\Tests\Unit\Strategy;
+
+use Exolnet\Bento\Bento;
+use Exolnet\Bento\Feature;
+use Exolnet\Bento\Strategy\LogicAnd;
+use Exolnet\Bento\Tests\UnitTest;
+use Mockery;
+
+class StrategyContainerTest extends UnitTest
+{
+    /**
+     * @var Exolnet\Bento\Bento
```

Comment:
Tu dois mettre le backslash devant Exolnet, sinon c'est comme si tu référençait la classe `Exolnet\Bento\Tests\Unit\Strategy\Exolnet\Bento\Bento`

```suggestion
     * @var \Exolnet\Bento\Bento
```